### DISCUSSION TITLE
Naming convention for tags

### DISCUSSION THREAD

**Discussion by cesarcoatl** [microsoft/azuredatastudio#14771] [General]
For some you use semantic versioning, and the next one will have either `_rel` or `_release` appended.

So I was wondering why it appears as if you did not have a standard for naming your releases.

**Comment by Charles-Gagnon** [comment]
Thanks for bringing this to our attention! There were some stale tags that need to be cleaned up - we'll be working on doing that and try to be more consistent in the future!

Out of curiosity - is this something that's affecting you or just something you noticed that seemed odd?

**Reply by cesarcoatl** [reply]
> Is this something that's affecting you or just something you noticed that seemed odd?

It is something I noticed on the Homebrew Cask for Azure Data Studio (https://github.com/Homebrew/homebrew-cask/pull/100237), where someone introduced a `livecheck` starting with version 1.25.2 (tag/1.25.2) that was relying on the tag following the X.X.X convention and it stopped working with the release of version 1.26.0 (tag/1.26.0_release). Since then I felt curious about this subject.

**Reply by Charles-Gagnon** [reply]
Alright, I went through and cleaned up all the old invalid tags and then made sure that we had tags following the x.x.x naming scheme for all our previous releases. Let me know if there's any other issues and thanks for bringing this up!
